stub_test/Makefile

```diff
@@ -21,21 +21,21 @@ clean:
 
 .PHONY: $(BUILDDIR)/test.native
 $(BUILDDIR)/test.native: $(BUILDDIR)/test.o
-	$(LD) -g -isystem ${GTEST} -o $@ $< -L$(BUILDDIR)/.. -lrelayout libgtest.a -pthread
+	$(LD) -keep_dwarf_unwind -pg -g -isystem ${GTEST} -o $@ $< -L$(BUILDDIR)/.. -lrelayout libgtest.a -pthread
 
 $(BUILDDIR)/%.o: %.cpp
-	$(CC) -g -c -o $@ -I $(OCAMLDIR) -I ../stub $<
+	$(CC) -pg -g -c -o $@ -I $(OCAMLDIR) -I ../stub $<
```

Comment:
`-pg` is for profiling. I hope that the android build tooling strips this? You may want to check that this actually works well for you.